libkineto/src/RoctracerLogger.cpp

```diff
@@ -410,7 +410,7 @@ void RoctracerLogger::stopLogging() {
   roctracer_stop();
 }
 
-void RoctracerLogger::endTracing() {
+void RoctracerLogger::endTracing() const {
```

Comment:
Hi @cyyever one more issue.
This function cannot be marked as const since its assigning the member "hccPool_" to nullptr.
This causes a build failure.

Reply:
Fixed